option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message RebelRaidStartRequest
{
 required U64Type serial_id = 1;
 required uint32 type = 2;
}
